server/monitor.go

```diff
@@ -1190,6 +1193,7 @@ func (s *Server) updateVarzRuntimeFields(v *Varz, forceUpdate bool, pcpu float64
 	v.Mem = rss
 	v.CPU = pcpu
 	v.Cores = numCores
+	v.MaxProcs = maxProcs
```

Comment:
Move this to createVarz() since I don't think that this value will change at runtime...